Documentation/ExtensionArchitecture/Extbase/Reference/FrontendPlugins.rst

```diff
@@ -43,10 +43,16 @@ Use the following steps to add the plugin as content element:
 
     Use the following parameters:
 
-    #. Extension key :php:`'blog_example'` or name :php:`BlogExample`.
-    #. A unique identifier for your plugin in UpperCamelCase: :php:`'PostSingle'`
-    #. An array of allowed combinations of controllers and actions stored in an array
-    #. (Optional) an array of controller name and  action names which should not be cached
+    #.  Extension key :php:`'blog_example'` or name :php:`BlogExample`.
+    #.  A unique identifier for your plugin in UpperCamelCase: :php:`'PostSingle'`
+    #.  An array of allowed combinations of controllers and actions stored in an array
+    #.  (Optional) an array of controller name and  action names which should not be cached
+    #.  Using any value but `ExtensionUtility::PLUGIN_TYPE_CONTENT_ELEMENT` is
```

Comment:
Only `list_type` will create a deprecation log entry

Reply:
And the default value which is empty but defaults to list_type as well. So for all practical reasons: just use ExtensionUtility::PLUGIN_TYPE_CONTENT_ELEMENT. I don't think details beyond this are needed